hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -404,13 +403,12 @@ private boolean hasRequiredReplicas(ContainerInfo contInfo) {
 
   @Override
   public void deleteContainer(long containerID) throws IOException {
-    String remoteUser = getRpcRemoteUsername();
     boolean auditSuccess = true;
     Map<String, String> auditMap = Maps.newHashMap();
     auditMap.put("containerID", String.valueOf(containerID));
-    auditMap.put("remoteUser", remoteUser);
+    auditMap.put("remoteUser", getRpcRemoteUsername());
     try {
-      getScm().checkAdminAccess(remoteUser);
+      getScm().checkAdminAccess(Server.getRemoteUser());
```

Comment:
Also note that `TestStorageContainerManager` integration test is failing, because `Server.getRemoteUser()` returns `null` (no mock set up for this static method, as opposed to `SCMClientProtocolServer.getRpcRemoteUsername()`), so access is granted.

https://github.com/apache/ozone/pull/2236/checks?check_run_id=2553193751#step:4:4223

Reply:
Fixed it